frontend/src/lib/components/Projects/ProjectFilter.svelte

```diff
@@ -71,6 +71,11 @@
           <AuthenticatedUserIcon />
           {filter.value}
         </ActiveFilter>
+      {:else if filter.key === 'hideDraftProjects'}
+        <ActiveFilter {filter}>
+          <Icon icon="i-mdi-script" />
+          {$t('project.filter.hide_drafts')}
+        </ActiveFilter>
```

Comment:
Nice. :-)

Reply:
You had already done the heavy lifting making the filter work. I couldn't resist making it visible 😉.
I went back to your `text-warning` color.

Reply:
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/4ac337b9-3724-4fb4-a684-a45cafd8d166)